src/Components/Publishing/Sections/StyledText.tsx

```diff
@@ -111,13 +112,15 @@ export const StyledText = div`
     margin: 0;
     word-break: break-word;
   }
-  .content-start {
-    ${Fonts.unica("s67", "medium")}
-    float: left;
-    line-height: .5em;
-    margin-right: 10px;
-    margin-top: .298em;
-    text-transform: uppercase;
+  p:first-child:first-letter {
```

Comment:
Wasn't aware of this selector, and it checks out here https://caniuse.com/#feat=css-first-letter 👍